artifacts/Music/Music.recipes

```diff
@@ -30,15 +30,45 @@ particle ShowArtist in 'source/ShowArtist.js'
       handle artist
   description `Learn more about ${artist}`
 
+particle ShowArtistSimple in 'source/ShowArtistSimple.js'
+  in Artist artist
+  consume item
+
 particle FilterToArtist in 'source/FilterToArtist.js'
   in ShowcaseArtist artist
   in [PlayRecord] fullPlayHistory
   inout [PlayRecord] artistPlayHistory
 
+particle ExtractArtists in 'source/ExtractArtists.js'
+  in [PlayRecord] fullPlayHistory
+  inout [RankedArtist {Text name, Number score}] artists
+
 particle PlayRecordRenderer in 'source/PlayRecordRenderer.js'
   in PlayRecord playRecord
   consume item
 
+recipe
+  map 'myNowPlayingHistory' as fullPlayHistory
```

Comment:
Fyi I'm removing the hardcoded thingy in #2033, but we can handle this in another PR.